### PR TITLE
[blockem] Cleanups:

### PR DESCRIPTION
**Pull request by Quix0r**
- used `use Friendica\App;`
- used proper type-hints
- added spaces and curly braces for better readability

### CODE REVIEW

**Comment by Quix0r** [comment]
If this way is okay for @MrPetovan I continue the same way with other addons.

File: blockem/blockem.php [hunk 1]
```diff
@@ -4,49 +4,52 @@
...
 {
-
-	if(! local_user())
+	if (! local_user()) {
 		return;
+	}
 
     /* Add our stylesheet to the page so we can make our settings look nice */
-
     $a->page['htmlhead'] .= '<link rel="stylesheet"  type="text/css" href="' . $a->get_baseurl() . '/addon/blockem/blockem.css' . '" media="all" />' . "\r\n";
```
Comment (MrPetovan) [inline blockem/blockem.php]: Indentation looks wrong (space instead of tabs)

File: blockem/blockem.php [hunk 2]
```diff
@@ -68,35 +71,37 @@ function blockem_addon_settings(&$a, &$s)
 
 }
 
-function blockem_addon_settings_post(&$a,&$b) {
+function blockem_addon_settings_post(App $a, array &$b) {
```
Comment (Quix0r) [inline blockem/blockem.php]: Oh, new-line character is missing?

File: blockem/blockem.php [hunk 3]
```diff
@@ -68,35 +71,37 @@ function blockem_addon_settings(&$a, &$s)
...
-	if($_POST['blockem-submit']) {
-		PConfig::set(local_user(),'blockem','words',trim($_POST['blockem-words']));
+	if ($_POST['blockem-submit']) {
+		PConfig::set(local_user(), 'blockem', 'words', trim($_POST['blockem-words']));
 		info(L10n::t('BLOCKEM Settings saved.') . EOL);
 	}
 }
 
-function blockem_enotify_store(&$a,&$b) {
+function blockem_enotify_store(App $a, array &$b) {
```
Comment (Quix0r) [inline blockem/blockem.php]: Oh, new-line character is missing?

File: blockem/blockem.php [hunk 4]
```diff
@@ -137,24 +142,25 @@ function blockem_prepare_body_content_filter(\Friendica\App $a, &$hook_data)
...
 
-function blockem_display_item(&$a,&$b) {
-	if (empty($b['output']['body'])) {
-		return;
-	}
-
-	if(strstr($b['output']['body'],'id="blockem-wrap-'))
+function blockem_display_item(App $a, array &$b = null)
+{
+	if (isset($b['output']) && strstr($b['output']['body'], 'id="blockem-wrap-')) {
```
Comment (MrPetovan) [inline blockem/blockem.php]: The first condition can be `!empty($b['output']['body'])` instead.

**Comment by MrPetovan** [comment]
Tat’s perfect. curweather indenting is all spaces for example.

File: blockem/blockem.php (hunk 1 above)
Comment (Quix0r) [reply blockem/blockem.php]: A lot lines will then be changed, if that is okay, I start changing?

Comment (MrPetovan) [reply blockem/blockem.php]: Yes, as long as it’s limited to a single addon 😉

**Comment by Quix0r** [comment]
Should be okay now.

**Pull request merged**